go.mod

```diff
@@ -1,13 +1,16 @@
 module github.com/pressly/goose/v3
 
-go 1.18
+go 1.20
 
 require (
 	github.com/ClickHouse/clickhouse-go/v2 v2.9.1
 	github.com/go-sql-driver/mysql v1.7.1
+	github.com/google/go-cmp v0.5.9
 	github.com/jackc/pgx/v5 v5.3.1
 	github.com/microsoft/go-mssqldb v0.21.0
 	github.com/ory/dockertest/v3 v3.10.0
+	github.com/stretchr/testify v1.8.2
```

Comment:
I avoid this package whenever possible, there is an internal one that should accomplish the same thing: 

https://github.com/pressly/goose/blob/3717a9e3157aef55a3907d8af1283c91cb78a17a/internal/check/check.go#L11

Reply:
Sure. I can update the PR to use the internal one.

Reply:
Fixed now.